workflows/nf_gwas.nf

```diff
@@ -28,6 +28,11 @@ run_gene_tests = params.regenie_run_gene_based_tests
 
 skip_predictions = params.regenie_skip_predictions
 
+if(params.regenie_interaction){
+    run_interaction = params.regenie_interaction
+} else {
+    run_interaction = params.regenie_interaction_snp
+}
```

Comment:
HI @johsr,
Is the idea here to set run_interaction to "true" in case if one of these parameters is true? If yes, maybe we could also write that like this and remove the if/else
```run_interaction = params.regenie_interaction || params.regenie_interaction_snp```

Reply:
Yes, exactly - sounds good thanks. I don't know if its relevant: regenie_interaction and regenie_interaction_snp are set to 'null' and can be set to certain values (the names of the interaction variables), so they are not by themselves set to true or false. Does it work anyways?

Reply:
you could init them to “false”

Reply:
Ok, these variables are strings. You could add an additonal option similar to `regenie_run_gene_based_tests` (e.g. `regenie_run_interaction_tests`) and use this variable [here](https://github.com/genepi/nf-gwas/blob/224e38996d675ca41fac1d77fe7eb227c1d06af4/workflows/nf_gwas.nf#L138)